src/entities/User.java

```diff
@@ -0,0 +1,89 @@
+package entities;
+
+import java.util.Objects;
+
+public class User {
```

Comment:
You may need `userame`, `password` as well.

Possibly you may need to add a Role to it. Since I'm not pretty sure about your idea could you document it in the description of this PR?